docs/2.0/docs/pipelines/installation/addingexistinggitlabrepo.mdx

```diff
@@ -0,0 +1,879 @@
+
+```bash
+mise use -g terragrunt@latest opentofu@latest
+```
+
+:::note Progress Checklist
+
+<PersistentCheckbox id="install-terragrunt-and-opentofu" label="Install Terragrunt and OpenTofu locally (if not already installed)." />
+
+:::
+
+### Cloud-specific bootstrap instructions
+
+<Tabs>
+<TabItem value="aws" label="AWS" default>
+
+The resources you need provisioned in AWS to start managing resources with Pipelines are:
+
+1. An OpenID Connect (OIDC) provider
+2. An IAM role for Pipelines to assume when running Terragrunt plan commands
+3. An IAM role for Pipelines to assume when running Terragrunt apply commands
+
+For every account you want Pipelines to manage infrastructure in.
+
+:::tip Don't Panic!
+
+This may seem like a lot to set up, but the content you need to add to your project is minimal. The majority of the work will be pulled from a reusable catalog that you'll reference in your project.
+
+If you want to peruse the catalog that's used in the bootstrap process, you can take a look at the [terragrunt-scale-catalog](https://github.com/gruntwork-io/terragrunt-scale-catalog) repository.
+
+:::
+
+The process that we'll follow to get these resources ready for Pipelines is:
+
+1. Use Boilerplate to scaffold bootstrap configurations in your project for each AWS account
+2. Use Terragrunt to provision these resources in your AWS accounts
+3. (Optionally) Bootstrap additional AWS accounts until all your AWS accounts are ready for Pipelines
+
+{/* We're using an h3 tag here instead of a markdown heading to avoid adding content to the ToC that won't work when switching between tabs */}
+<h3>Bootstrap Your Project for AWS</h3>
+
+First, confirm that you have a `root.hcl` file in the root of your project that looks something like this:
+
+```hcl title="root.hcl"
+locals {
+  account_hcl       = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+  state_bucket_name = local.account_hcl.locals.state_bucket_name
+
+  region_hcl = read_terragrunt_config(find_in_parent_folders("region.hcl"))
+  aws_region = local.region_hcl.locals.aws_region
+}
+
+remote_state {
+  backend = "s3"
+  generate = {
+    path      = "backend.tf"
+    if_exists = "overwrite"
+  }
+  config = {
+    bucket       = local.state_bucket_name
+    region       = local.aws_region
+    key          = "${path_relative_to_include()}/tofu.tfstate"
+    encrypt      = true
+    use_lockfile = true
+  }
+}
+
+generate "provider" {
+  path      = "provider.tf"
+  if_exists = "overwrite_terragrunt"
+  contents  = <<EOF
+provider "aws" {
+  region = "${local.aws_region}"
+}
+EOF
+}
+```
+
+If you don't have a `root.hcl` file, you might need to customize the bootstrapping process, as the Terragrunt scale catalog expects a `root.hcl` file in the root of the project. Please contact [Gruntwork support](/support) for assistance if you need help.
+
+For each AWS account that needs bootstrapping, we'll use Boilerplate to scaffold the necessary content.
```

Comment:
Might be missing an instruction here to tell the user to run this boilerplate command in _each account folder_ or similar?